CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ FetchContent_Populate(folly)
 
 add_library(folly_headers INTERFACE)
 target_include_directories(folly_headers SYSTEM INTERFACE ${folly_SOURCE_DIR})
+target_link_libraries(folly_headers INTERFACE Boost::headers)
```

Comment:
Do you know why this is needed now? There don't seem to be any references to boost in the old Makefile

Reply:
I think CMake is doing the dependency management properly here. Boost is in `/usr/include/boost/` for me so a C++ compiler would be able to find it without any extra paths, but CMake seems to want it explicitly specified. It definitely fails to build with just Folly headers as `folly::Functional` depends on `expr_iif` or something in Boost.